docs/components.md

```diff
@@ -1,12 +1,12 @@
 Components
 ==========
 
-Think in components. Consider of each piece of your UI as an individual "component."
+Think in components. Consider each piece of your UI as an individual "component".
```

Comment:
`"xxx."` is commonly preferred but I guess I'll allow it. http://www.grammarbook.com/punctuation/quotes.asp